omero/sysadmins/unix/server-centos6.txt

```diff
@@ -220,36 +212,18 @@ As the **omero** system user, configure OMERO.web by running the following comma
 The version of Apache installed is 2.2, see the note below if you wish to use
 version 2.4.
 
-To install Apache 2.2 run::
+Install Apache 2.2, install the requirements to run OMERO.web in the virtual environment, deactivate it
+and add the path to the virtual environment python to the ``python-path`` parameter of the ``WSGIDaemonProcess`` directive::
 	
```

Comment:
Same issue as above, the double `::` is making the code box contain the directive command rather than the script text.